libs/glib2/Makefile

```diff
@@ -46,7 +45,7 @@ define Package/glib2/description
   The GLib library of C routines
 endef
 
-TARGET_CFLAGS += -Wno-error=implicit-function-declaration
+TARGET_CFLAGS += $(FPIC) -ffunction-sections -fdata-sections -flto -Wno-error=implicit-function-declaration
```

Comment:
Honestly, I feel like this is a bigger change than somethign to sneak in under "fix mips16 builds"

Reply:
"sneak" ok, i can change the commit text to include the flags change?

Reply:
-Wno-error=implicit-function-declaration can be removed. It's left over from a previous version.

--disable-compile-warnings to CONFIGURE_ARGS disables -Werror.